website/docs/components/drag-and-drop/drag-and-drop.md

```diff
@@ -9,117 +9,60 @@ tabs: Design('drag-and-drop'), A11y('drag-and-drop-a11y'), API('drag-and-drop-ap
-| Default  | ![](static/droppable-zone.png)                                              | By default, the area where the element can be placed is marked with a dashed border. You can place text inside this area to describe what can be dropped there, using the secondary text color (`--text-secondary`) that is appropriate for the context. |
-| Hover    | ![](static/default-and-hover.png)                                           | The border color of the area where an object can be dragged isn’t changed when hovered over.                                                                                                                                                                                   |
-| dragging | ![](static/drag-card.png) ![](static/drag-table2.png) | Hovering over the area where an element can be dropped changes its color. When dragging an object in a list of similar objects, they are separated, and the drop zone changes the background color to `--bg-secondary-neutral-hover`.       |
-| no-drop  | ![](static/no-drop.png)                                                            | If the drop zone under the dragged object is unavailable, the cursor changes to `not-allowed`.                                                                                                                                                                             |
-
-## Usage in UX/UI
-
-This component imitates physically dragging objects, so keep the following in mind when implementing it:
-
-- Visually indicate that the element is draggable using icons or text.
-- Display the area where the element can be placed.
-- Avoid placing the draggable element and its drop area too far apart.
-
-::: tip
-Use appropriate cursors such as `move`, `grab`, or `grabbing` when interacting with draggable elements.
-:::
+| State description | Illustration |
+|-------------------|--------------|
+| In the default state, the drop zone is marked with `border: 1px dashed var(--border-primary)`. Display text with `--text-secondary` color inside the zone to describe what can be dropped there. <br/><br/>When nothing is dragged, the drop zone isn't interactive and its styles don't change on hover. | ![](static/drop-zone-default.png)  |
+| When an object is dragged over the drop zone, if the object was dragged from another area, the drop zone background changes to `--bg-primary-neutral-hover`.  | ![](static/drag-card.png)  |
+| When rearranging objects in a list or grid, the drop zone is created dynamically by pushing apart neighbouring objects. In this case, the drop zone background is `--bg-secondary-neutral-hover`.  | ![](static/dragging.png)  |
+| If the drop zone under the dragged object is unavailable, mouse cursor changes to `not-allowed`.  | ![](static/no-drop.png)  |
 
+<!--
 Note that short and long clicks have different meanings, with long clicks used for clickable draggable objects.
```

Comment:
this sounds like objects can be grabbed by their entire area. If we want to keep this, I'll need to rewrite this somewhat.